examples/asr/conf/lang_id/titanet_large.yaml

```diff
@@ -145,12 +144,10 @@ model:
     num_classes: 107
     pool_mode: 'xvector' # 'attention'
     emb_sizes: 512
-    angular: False
 
   loss:
-    weight: 'auto' # could either be 'auto' or 1D tensor
-    scale: 30 # specify this if use angular loss
-    margin: 0.2  # specify this if use angular loss
```

Comment:
can you also update gradient_clip_val to 1.0

Reply:
Sure.